src/purify.js

```diff
@@ -900,19 +900,18 @@ function createDOMPurify(window = getGlobal()) {
   };
 
   /**
-   * _createIterator
+   * Creates a NodeIterator object that you can use to traverse filtered lists of nodes or elements in a document.
    *
-   * @param  {Document} root document/fragment to create iterator for
-   * @return {Iterator} iterator instance
+   * @param  {Node} root The root element or node to start traversing on.
+   * @return {NodeIterator} The created NodeIterator
    */
-  const _createIterator = function (root) {
+  const _createNodeIterator = function (root) {
```

Comment:
https://github.com/microsoft/TypeScript/blob/main/src/lib/dom.generated.d.ts#L7160
I've modified the jsdoc with reference to that documentation.

Also, the naming seems clearer for createNodeIterator than createIterator.